server/src/main/java/org/opensearch/action/admin/cluster/node/tasks/create/CreateTaskResponse.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.node.tasks.create;
+
+import org.opensearch.common.annotation.PublicApi;
+import org.opensearch.core.action.ActionResponse;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+import org.opensearch.tasks.Task;
+
+import java.io.IOException;
+
+/**
+ * Returns a new task.
+ *
+ * @opensearch.api
+ */
+@PublicApi(since = "1.0.0")
```

Comment:
why does it need to be PublicAPI and not internal. Applies for other classes as well

Reply:
yes, i will make it internal only.